android/res/xml/settings_application.xml

```diff
@@ -17,11 +17,11 @@
  */
 -->
 <!-- see ApplicationPreferencesFragment -->
-<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
-    <androidx.preference.PreferenceCategory
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
why move it back to PreferenceScreen? isn't androidx the new way?

Reply:
AndroidX is a major improvement to the original Android Support Library. Like the Support Library, AndroidX ships separately from the Android OS and provides backwards-compatibility across Android releases. AndroidX fully replaces the Support Library by providing feature parity and new libraries. In addition AndroidX includes the following features:

All packages in AndroidX live in a consistent namespace starting with the string androidx. The Support Library packages have been mapped into corresponding androidx.* packages. For a full mapping of all the old classes and build artifacts to the new ones, see the Package Refactoring page.

Reply:
We went for the old .v7 PreferenceScreen to the new stable androidx API 14 months ago
https://github.com/frostwire/frostwire/commit/76e522825eda2cab7a16045f484c563b8b552eac#diff-579144e5114ee0bc6d4a0fb854995a8a

Reply:
I see, now `PreferenceScreen` is implicitly `androidx`'s

Merging. Works all the way down to Android 5.0